rskj-core/src/main/java/co/rsk/rest/modules/RestModule.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * This file is part of RskJ
+ * Copyright (C) 2018 RSK Labs Ltd.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+package co.rsk.rest.modules;
+
+import io.netty.handler.codec.http.DefaultFullHttpResponse;
+import io.netty.handler.codec.http.HttpMethod;
+
+public abstract class RestModule {
```

Comment:
Here we could add an api rest method list to be sure that all the modules are defining them, and a maybe generic method to validate the uri and the rest method, as it would be the same for the modules.

Reply:
The logic of `URI` comparison and request dispatching is done at `Dispatcher` level, by evaluating the base URI and module status (active/inactive).